### PR TITLE
Frontend finalize

### PR DESCRIPTION
**Pull request by aqsa505**

### CODE REVIEW

**Comment by aqsa505** [comment]
> Line 12 ends with , which seems like an error. That should be removed if it is, since is in the following line (line 13).

Line 12 of index.html?

File: cards/templates/cards/index.html [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+<body style="background-image: url('https://i0.wp.com/bullishcoder.com/wp-content/uploads/2019/06/creditCard.jpg?w=800&ssl=1'); background-size: 100% 100%; background-opacity: 0.1;">
+
   <div class="container">
-    <br>
+      <br>
   <h1 class = "display-4" style = "text-align: center;">We'll help choose the best credit card for you</h1>
   <p class= "font-weight-normal" style = "text-align: center; font-size: 20px; color: #595957;">Are you undecided on a particular credit card? Are you looking for better rewards?</p>
   <p class="font-weight-normal" style = "text-align: center; font-size: 20px;">WhichCard allows you to compare different rewards credit cards based on your spending habits. 
     <br>We can even help novice credit card users earn the most points, mileage or cash back with our recommendations.</p>
   <h3 style="color:#337CFF; text-align: center;">Instantly discover your best credit card options</h5>
```
Comment (NComer98) [inline cards/templates/cards/index.html:12]: Yeah.

**Comment by NComer98** [comment]
Right. It starts with the h3 tag.

**Comment by jvberdec** [comment]
I saw the home page and it is difficult to read the text. It looks incredibly flat and visually unappealing- I was going for a much brighter look that matches the overall theme. I am afraid that the home page will look cheap. This is the first page that a user would see when entering our site and I think we can do much better.

**Comment by aqsa505** [comment]
I think we should change the font size and font style

File: cards/templates/cards/AboutUs.html [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
     <h5 style="color:#337CFF; font-size:30px;">Our Product</h5>
-    <p class="font-weight-normal">We’ll develop a credit card comparison tool that accounts for peoples’ spending habits, and instantly ranks credit card options from America’s top 6 banks - from most to least rewarding. The tool will also display to users their first year rewards value estimate, and provide a few details about each card so they can quickly compare cards side-by-side.
+    <p class="font-weight-normal">We developed a credit card comparison tool that accounts for peoples’ spending habits and instantly ranks credit card options from America’s top 6 banks - from most to least rewarding. The tool will also display to users their first year rewards value estimate, and provide a few details about each card so they can quickly compare cards side-by-side.
         
         <p class="font-weight-normal"></p>We can collect information on credit cards from here: 
         <a href="https://www.creditcards.com/rewards-cards/">creditcards.com-Card Rewards</a>
```
Comment (natharrow) [inline cards/templates/cards/AboutUs.html:16]: We can remove line 15 and 16 - the link to creditcards.com. It was just info for us when we started the project. I. will remove it from our README as well

Comment (jvberdec) [reply cards/templates/cards/AboutUs.html:16]: Gotcha! Thanks.

**Pull request merged**